packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res

```diff
@@ -124,6 +124,20 @@ module Beta = {
   let sample = (t: t) => Jstat.Beta.sample(t.alpha, t.beta)
   let mean = (t: t) => Ok(Jstat.Beta.mean(t.alpha, t.beta))
   let toString = ({alpha, beta}: t) => j`Beta($alpha,$beta)`
+
+  let fromMeanAndSampleSize = (mean, sampleSize) => {
+    // https://en.wikipedia.org/wiki/Beta_distribution#Mean_and_sample_size
+    let alpha = mean *. sampleSize
+    let beta = (1.0 -. mean) *. sampleSize
+    alpha->make(beta)
+  }
+
+  let fromMeanAndStdev = (mean, stdev) => {
+    // https://en.wikipedia.org/wiki/Beta_distribution#Mean_and_variance
+    let var = stdev *. stdev
+    let sampleSize = mean *. (1.0 -. mean) /. var -. 1.0
```

Comment:
I think this is might only be valid where:
``var < mean ( 1 - mean)``
... according to the wikipedia page?

Reply:
Yes, it might be clearer to have a check here. But note that if the make is called with invalid parameters in line 139 (<https://github.com/quantified-uncertainty/squiggle/blob/5b41019c04d4b5137af7776bc21b84e784635371/packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res#L139=>), it will be caught by the test in line 118 (<https://github.com/quantified-uncertainty/squiggle/blob/5b41019c04d4b5137af7776bc21b84e784635371/packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res#L118=>)

Reply:
I see that Quinn already pointed this out above, but still in favour of catching errors earlier.